modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -63,6 +65,22 @@ imageContentSourcePolicy.yaml
 mapping.txt
 ----
 
+. Create a CatalogSource object for the new catalog:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: my-operator-catalog
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: <registry_host_name>:<port>/<namespace>/redhat-operators:v1
```

Comment:
It's better if we highlight this image, such as: create a CatalogSource referencing the image you created above.